pandas/core/generic.py

```diff
@@ -4983,16 +4980,13 @@ def pipe(self, func, *args, **kwargs):
         * DataFrame : when DataFrame.agg is called with several functions
 
         Return scalar, Series or DataFrame.
-
     %(see_also)s
-
     Notes
     -----
     `agg` is an alias for `aggregate`. Use the alias.
 
     A passed user-defined-function will be passed a Series for evaluation.
-
-    %(examples)s
+    %(examples)s\
```

Comment:
So I think generally we just need to figure out one way of doing things. Maybe we change this down the road but as far as this PR goes would rather just move all of the triple quote terminators up a line rather than putting in the escape character, just because it's not very apparent why the escape might be there to future readers